app/code/Magento/Store/Model/StoreManager.php

```diff
@@ -307,7 +345,11 @@ private function getStoreWebsiteRelation()
     }
 
     /**
-     * @inheritdoc
+     * Get assigned to website store
```

Comment:
Please revert the comment and just remove curly braces `{}`. Please refer to the guide https://devdocs.magento.com/guides/v2.3/coding-standards/docblock-standard-general.html#inheritdoc